pkg/deploy/predeploy.go

```diff
@@ -26,8 +26,8 @@ import (
 )
 
 // Rotate the secret on every deploy of the RP iff the most recent
```

Comment:
nit: while we are here, mind fixing the typo in the word `iff`?

Reply:
secondly - isn't the comment backwards
"rotate if most recent is less than 7 days" -> don't we only rotate if most recent is greater than 7 days?

Reply:
Is `iff` a typo or is it just not used properly ?

Reply:
> secondly - isn't the comment backwards "rotate if most recent is less than 7 days" -> don't we only rotate if most recent is greater than 7 days?

Yes, exactly. If the delay between the now() of the predeploy and the previous pre-deploy which rotated the secret, then rotate.

Reply:
> Is `iff` a typo or is it just not used properly ?

I agree, this is not an "iff" but an "if". The condition is necessary but not sufficient :)

Reply:
iff -> if and only if

But the above isn't necessarily true in this case as we still rotate if a secret isn't present.  Fixed.